lambda-alb/src/AWS/Lambda/ALB.hs

```diff
@@ -73,13 +81,33 @@ data Request a = Request
   deriving stock    (Generic, Show, Eq)
 
 data Response = Response
-  { isBase64Encoded :: Bool
-  , statusCode      :: Natural
-  , headers         :: Headers
-  , body            :: Text
+  { statusCode :: HTTP.Status
+  , headers    :: Headers
+  , body       :: ResponseBody
```

Comment:
We need to be able to efficiently compute the content length of the HTTP response body in the ALB. Before the ALB response gets turned into a byte string by the ALB. 

If we where to store the response body as a text, the computation from XRay would have to serialize to UTF-8 encoded bytestring just to count the bytes. 

This API allows to later add `Response toEncoding` allowing us to spare the extra serialization. For the moment I do not care about efficiency that much so delaying the `toEncoding`. But still made the API so its easy to add later without any breakage.